docs/pipeline/rules/headerscollection.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Headers Collection"
+description: "The \"Headers Collection\" rule can be used to add span attributes by collecting HTTP Headers data."
+sidebarTitle: "Headers Collection"
+icon: "webhook"
+---
+
+<Info>
+  This Instrumentation Rule is currently only available with the Odigos **Enterprise** plan.<br />
+  [Contact us](https://odigos.io/) for more information.
+</Info>
+
```

Comment:
for http we have both client/server, and request/response.

I guess this configuration option will apply to all of them? might be worth mentioning here